docker-compose.yml-example

```diff
@@ -320,3 +323,6 @@ networks:
   web:
   kopanonet:
     driver: bridge
+  ldap-net:
+    name: ldap-net
```

Comment:
Since this network has a dash in its name, why not rename the other network to have a dash as well? Instead of "web" then also "public-web"?

Reply:
Good point. I will update those.